addons/account/wizard/setup_wizards.py

```diff
@@ -99,9 +114,12 @@ def set_linked_journal_id(self):
                 })
             else:
                 selected_journal.bank_account_id = record.res_partner_bank_id.id
+                selected_journal.name = record.new_journal_name
+                selected_journal.code = record.new_journal_code
+            selected_journal.mark_bank_setup_as_done_action()
```

Comment:
Why not doing that in validate() ?

Reply:
Because it is part if the computation of set_linked_journal_id, I'm not sure what you mean

Reply:
I'm talking about the call to mark_bank_setup_as_done_action ;)